lib/nuntiux/mocker.ex

```diff
@@ -10,10 +10,9 @@ defmodule Nuntiux.Mocker do
            opts: Nuntiux.opts()
          }
 
-  @doc """
-  Boots up a mocking process.
-  If the process to be mocked doesn't exist, returns <pre>ignore</pre>.
-  """
```

Comment:
I consider this an internal implementation detail, so prefer to not have it in the doc.